docs/azure_monitor/export/export.options.rst

```diff
@@ -0,0 +1,9 @@
+Exporter Options
+================
+
+
+
+.. autoclass:: azure_monitor.options.ExporterOptions
```

Comment:
`__slots__` looks strange in the docs.

![image](https://user-images.githubusercontent.com/11580155/77374965-6099cb80-6d29-11ea-9c22-5361be251b2f.png)